tests-functional/tests/test_wakuext_community_token_permissions.py

```diff
@@ -0,0 +1,246 @@
+import logging
+import time
+from typing import Optional, List
+
+import pytest
+
+from clients.contract_deployers.snt import SNTDeployer
+from clients.services.wakuext import CommunityPermissionsAccess, CommunityTokenPermissionType, CommunityTokenType, CommunityRoles
+from clients.signals import SignalType
+from resources.constants import user_1
+from resources.enums import RequestToJoinState
+from steps.messenger import MessengerSteps
+from utils import fake
+from utils.retry_utils import retry_call
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.rpc
+class TestCommunityTokenPermissions(MessengerSteps):
+    def _communities_list(self, communities_response):
+        if isinstance(communities_response, dict):
+            return communities_response.get("communities", []) or []
+        return communities_response or []
+
+    @pytest.fixture(autouse=True)
+    def setup_backends(self, backend_new_profile, foundry_client):
+        """Initialize backends for token permission tests"""
+        self.owner = backend_new_profile("owner")
+        self.member = backend_new_profile("member")
+        self.non_member = backend_new_profile("non_member")
+        self.fake_address = "0x" + "0" * 40  # Fake address for testing
```

Comment:
This seem to be used only in one place, let's define it there.

Reply:
Moved at https://github.com/status-im/status-go/pull/7113/commits/7e1bb4f5620d53041222ebca9f1283f7a86e0383